osutil/mountinfo_linux.go

```diff
@@ -91,12 +93,19 @@ func (mi *MountInfoEntry) String() string {
 		flattenMap(mi.SuperOptions))
 }
 
-// LoadMountInfo loads list of mounted entries from a given file.
-//
-// The file is typically ProcSelfMountInfo but any other process mount table
-// can be read the same way.
-func LoadMountInfo(fname string) ([]*MountInfoEntry, error) {
-	f, err := os.Open(fname)
+// LoadMountInfo loads list of mounted entries from /proc/self/mountinfo. This
+// can be mocked by using osutil.MockMountInfo to hard-code a specific mountinfo
+// file content to be loaded by this function
+func LoadMountInfo() ([]*MountInfoEntry, error) {
+	if mockedMountInfo {
```

Comment:
this would become  mockedMountInfo != nil  and below *mockedMountInfo

Reply:
done